api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  const xAccessToken = req.get('X-Access-Token');
+  if (!xAccessToken) {
+    throw new ClientError(401, 'authentication required');
+  }
+  try {
+    const payload = jwt.verify(xAccessToken, process.env.TOKEN_SECRET);
+    console.log(payload);
+    req.user = payload
+    next();
+  } catch (err) {
+    console.error(err);
+  }
```

Comment:
You don't have to use a `try catch` here but since you did you have to call `next` in the `catch` block and pass it the error object.
```JAVASCRIPT
catch (err) {
  next(err);
}
```

Reply:
Thanks for feedback @scbowler. I removed the try catch statement. let me know if this works still.

Reply:
Yep, that will do it!